lars-reimann/api-editor
# fix: valid URL after package import

Closes #147.

## Summary of Changes
Navigates to the root URL after a package is imported.

## Testing instructions
1. Select something in the tree view.
2. Import a new package.
3. Observe the URL.
